apps/partner2/client/overview_layout_factory.coffee

```diff
@@ -16,7 +16,14 @@ articlesTemplate = -> require('../components/fixed_cells_count_carousel/articles
 
 module.exports = (partner, profile) ->
   contract =
-    institution: []
+    institution: [
+      galleryThreeHero partner, profile
+      galleryThreeAbout partner, profile
+      galleryThreeNews partner, profile
+      galleryThreeShows partner, profile
+      galleryThreeFairBooths partner, profile
+      galleryThreeArticles partner, profile
+    ]
```

Comment:
Also just chatted to @garrengotthardt about the artists tab a bit and he clarified that the layout for Institutions should be based on `PREFERRED` partners which is actually `galleryTwo` in our terminology. Should be updated here.